content/html/concepts/elements/terms/track/track.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'track' 
+Description: 'An HTML element that specifies subtitles, closed captioning, and other text files for the `<audio>`or the `<video>` element.' 
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+  
+Tags: 
+  - 'Attributes'
+  - 'Elements'
+  - 'Style'
+CatalogContent: 
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<track>`** element used to define and add the `<video>` or the `<audio>` elements' subtitles, closed captions, or other text-based content to an HTMl file. 
+## Syntax
+
+```pseudo
+<element 
+source= src="url">
+<track src="text-based file description" kind="type of text" 
+ other attribute="description"> 
+ </element>
+```
+`<track>` can be paired with the following attributes:
+  | HTML Attribute | Description |
+|---|---|
+| `chapter` | Utlized when user(s) navigates through the audio or film.|
+| `default` | Implies that the track should be enabled unless the user's preferences indicate that another track is best suited for the project.  **NOTE:** Can only be used with **one** `<track>` element at a time. |
```

Comment:
```suggestion
| `default` | Implies that the track should be enabled unless the user's preferences indicate that another track is best suited for the project. Can only be used with **one** `<track>` element at a time. |
```

Reply:
I would refrain from using the note in bold within a table. Generally, this is used relatively sparingly and below a section instead of inline. Please remove the other instances of "note" in this table.